public/assets/images/contentful-fragment-logo.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="239px" height="56px" viewBox="0 0 239 56" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Artboard" transform="translate(-80.000000, -43.000000)" fill-rule="nonzero">
+            <g id="Group-4" transform="translate(80.000000, 42.000000)">
```

Comment:
this `<g>` is actually unnecessary, since all it's doing is performing a translation.
```
-80 + 80 = 0
-43 + 43 = -1
```
soo line 3 will now read
```
<g id="Artboard" transform="translate(0, -1)" fill-rule="nonzero">
```